test/compliment/sources/t_class_members.clj

```diff
@@ -10,6 +10,39 @@
 
 (def ^Thread thread (Thread.))
 
+(deftest class-member-symbol-test
+  (fact "clojure <1.12"
+        (src/class-member-symbol? "")
+        => nil
+
+        (src/class-member-symbol? "a")
+        => nil
+
+        (src/class-member-symbol? "a/")
+        => nil
+
+        (src/class-member-symbol? ".")
+        => (just [nil ""])
+
+        (src/class-member-symbol? ".a")
+        => [nil "a"])
+
+  (fact "clojure 1.12+"
+        (src/class-member-symbol? "" {:clj-1-12+ true})
+        => nil
+
+        (src/class-member-symbol? "a" {:clj-1-12+ true})
+        => nil
+
+        (src/class-member-symbol? "a./" {:clj-1-12+ true})
+        => nil
+
+        (src/class-member-symbol? "a.b/" {:clj-1-12+ true})
+        => (just ["a.b" nil])
+
+        (src/class-member-symbol? "a.b/." {:clj-1-12+ true})
```

Comment:
Instead of doing this and the rebinding below, just limit these tests to being on Clojure 1.12+.

Reply:
By adding e.g `:exec-args {:includes [:1.12]}` to the 1.12 alias and earmarking the deftest?

Reply:
`(when (clojure-1-12+?) ...)` around the `fact` would be the easiest to do.